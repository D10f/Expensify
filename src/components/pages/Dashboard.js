import React from 'react'
import ExpenseList from '../ExpenseList'
import ExpenseListFilters from '../ExpenseListFilters'

export const Dashboard = () => (
  <section>
    <header>These are your current expenses:</header>
    <ExpenseListFilters />
    <ExpenseList />
  </section>
)

export default Dashboard
