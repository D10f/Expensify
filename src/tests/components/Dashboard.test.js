import React from 'react'
import { shallow } from 'enzyme'
import Dashboard from '../../components/pages/Dashboard'

test('should render Dashboard correctly', () => {
  const wrapper = shallow(<Dashboard />)
  expect(wrapper).toMatchSnapshot()
})
