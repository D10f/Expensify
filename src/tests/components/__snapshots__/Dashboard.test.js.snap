// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Dashboard correctly 1`] = `
<section>
  <header>
    These are your current expenses:
  </header>
  <Connect(ExpenseListFilters) />
  <Connect(ExpenseList) />
</section>
`;
